### DISCUSSION TITLE
Nominate Jerico Pena to the builders contributors team

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#206] [Builder Team]
I'd like to nominate Jerico Pena (@jericop) as a builders contributor. Jerico has provided valuable perspective on the experience of running on custom stacks and builders, and has contributed a lot to the discussion of multi-arch support. Jerico is willing and able to continue to provide input in the builders domain.

As Jerico starts to implement arm64 in the builders, the feedback loop will be much faster for him if is a contributor (no need for first-time PR approvals, etc).

@paketo-buildpacks/builders-maintainers can I get a 👍 / 👎 ?

**Comment by robdimsdale** [comment]
@paketo-buildpacks/steering-committee can we add Jerico to the builders contributors teams please?

**Reply by ForestEckhardt** [reply]
He has been invited could you add him to the team in the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md).

**Reply by robdimsdale** [reply]
https://github.com/paketo-buildpacks/community/pull/153
